modules/zip/src/lib/tar/utils.ts

```diff
@@ -5,36 +5,23 @@
  * MIT (c) 2011 T. Jameson Little
  */
 
-/**
```

Comment:
Instead of removing headers try to add a meaningful string, or if not clear just remove the type. The goal is to have a comment on **every** function.

Reply:
> Instead of removing headers try to add a meaningful string, or if not clear just remove the type. The goal is to have a comment on **every** function.

ok, it's starting to get clearer, the whole structure I mean

I made the second commit, would you mind to check those fixes?